src/player-wrapper.js

```diff
@@ -455,6 +455,19 @@ PlayerWrapper.prototype.onAdError = function(adErrorEvent) {
   }});
 };
 
+/**
+ * Handles ad log messages.
+ * @param {google.ima.AdEvent} adEvent The AdEvent thrown by the IMA SDK.
+ */
+PlayerWrapper.prototype.onAdLog = function(adEvent) {
+  const adData = adEvent.getAdData();
+  if (adData['adError']) {
```

Comment:
We call this adlog, but we only propagate it if it's an error inside of a log. We should either rename it (nonFatalError or something) or just propagate all LOG messages and let the user check to see if it's an error or not.

Reply:
I made it pass along the entire LOG message, in addition to the error message, if present.